### PR TITLE
Move census permission into access groups

### PR DESCRIPTION
**Pull request by eanders**
This needs to be deployed after https://github.com/greenriver/hmis-warehouse/pull/1315 since it requires the seeds provided by https://github.com/greenriver/hmis-warehouse/pull/1315.

### CODE REVIEW

File: db/migrate/20210127180420_move_dashboard_permissions.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      'dashboards/adult_only_households',
+      'dashboards/adults_with_children',
+      'dashboards/clients',
+      'dashboards/child_only_households',
+      'dashboards/non_veterans',
+      'dashboards/veterans',
+      'censuses',
+    ]
+    census_reports = GrdaWarehouse::WarehouseReports::ReportDefinition.where(url: urls)
+    User.joins(:roles).merge(Role.where(can_view_censuses: true)).distinct.select(:id, :email).find_each do |user|
```
Comment (awisema) [inline db/migrate/20210127180420_move_dashboard_permissions.rb:16]: Check to make sure `can_view_censuses` exists?

File: drivers/adult_only_households_sub_pop/app/controllers/adult_only_households_sub_pop/dashboards/adult_only_households_controller.rb [hunk 2]
```diff
@@ -7,8 +7,7 @@
 module AdultOnlyHouseholdsSubPop::Dashboards
   class AdultOnlyHouseholdsController < ::Dashboards::BaseController
     include ArelHelper
-
-    before_action :require_can_view_censuses!
+    include WarehouseReportAuthorization
```
Comment (awisema) [inline drivers/adult_only_households_sub_pop/app/controllers/adult_only_households_sub_pop/dashboards/adult_only_households_controller.rb]: Does this need to be here if it is in the base class?

**Review by awisema — CHANGES_REQUESTED** [review]

File: drivers/adult_only_households_sub_pop/app/controllers/adult_only_households_sub_pop/dashboards/adult_only_households_controller.rb (hunk 2 above)
Comment (eanders) [reply drivers/adult_only_households_sub_pop/app/controllers/adult_only_households_sub_pop/dashboards/adult_only_households_controller.rb]: Probably not

**Review by awisema — APPROVED** [review]

**Comment by eanders** [comment]
Closing as this doesn't seem necessary any more.

**Pull request closed without merging**